api/tests/python/models/test_request.py

```diff
@@ -107,6 +110,36 @@ def test_has_consumed_name():
 
     assert nr.has_consumed_name is True
 
+@pytest.mark.parametrize(['compName', 'compState', 'nrNum','nrEntity','assertValue'], [
+    ('TEST1', 'APPROVED','NR 0000001', 'FR', False),
```

Comment:
nice table test!
I'd only suggest giving a **test_name**, for each to give a hint as to what it is testing.

Reply:
thanks Thor, will do that.   great suggestion.

Reply:
added testName, and repushed